hashes/src/util.rs

```diff
@@ -194,21 +194,6 @@ macro_rules! hash_newtype {
 
         #[allow(unused)] // Private wrapper types may not need all functions.
         impl $newtype {
-            /// Creates this wrapper type from the inner hash type.
-            pub const fn from_raw_hash(inner: $hash) -> $newtype {
-                $newtype(inner)
-            }
-
-            /// Returns the inner hash (sha256, sh256d etc.).
-            pub const fn to_raw_hash(self) -> $hash {
-                self.0
-            }
-
-            /// Returns a reference to the inner hash (sha256, sh256d etc.).
-            pub const fn as_raw_hash(&self) -> &$hash {
-                &self.0
-            }
```

Comment:
I agree with the change but not justification. I think casting raw hashes is perfectly fine. The reason to remove it is to help getting 1.0 out sooner. Hopefully one day we can add it back.

Reply:
Personally I think that casting raw hashes should be done by `to_byte_array` and `from_byte_array` and we shouldn't have blessed "raw hash" versions.

But sure, let's default to not having them, and it's easy to add later if we agree that we should.